src/styles/_card.scss

```diff
@@ -0,0 +1,132 @@
+@use 'variables';
+
+.card {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  color: variables.$mainColor;
+
+  box-sizing: border-box;
+  width: 200px;
+  height: 408px;
+  padding-inline: 16px;
+  padding-block: 32px 16px;
+
+  border: 1px solid variables.$cardBorderColor;
+
+  border-radius: 5px;
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+
+    margin-inline: 3px;
+  }
+
+  &__title {
+    margin-top: 40px;
+  }
+
+  &__code {
+    margin-top: 4px;
+
+    font-family: Roboto, sans-serif;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: variables.$productCodeColor;
+  }
+
+  // #region review
+  &__review {
+    margin-top: 16px;
+
+    height: 16px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__review-stars {
+    width: 96px;
+    display: flex;
+    justify-content: space-between;
+  }
+
+  &__review-text {
+    width: 56px;
+
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    text-align: right;
+  }
+
+  // #endregion
+
+  // #region price
+  &__price {
+    margin-top: 24px;
+
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__price-text {
+    font-family: Roboto, sans-serif;
+    font-weight: 400;
+    text-align: left;
+    color: variables.$productCodeColor;
+  }
+
+  &__price-numbers {
+    font-family: Roboto, sans-serif;
+    font-weight: 700;
+    font-size: 16px;
+    text-align: right;
+  }
+
+  // #endregion
+
+  // #region button-buy
+  &__buy {
+    box-sizing: border-box;
+    display: block;
+    margin-top: 16px;
+    width: 166px;
+    height: variables.$buyButtonHeight;
+
+    line-height: variables.$buyButtonHeight;
+    font-size: 14px;
+    text-align: center;
+    text-decoration: none;
+    text-transform: uppercase;
+    color: variables.$buyTextColor;
+
+    background-color: variables.$secondaryColor;
+    border-radius: 5px;
+    border: 1px solid variables.$secondaryColor;
+
+    @include variables.transition;
+
+    &:hover {
+      background-color: variables.$buyTextColor;
```

Comment:
Ensure that `variables.$buyTextColor` is defined in the `variables` file. If it's not defined, this will cause an error during compilation.